I18n/backOffice/default/fr_FR.php

```diff
@@ -2,16 +2,23 @@
 
 return array(
     'All' => 'Tous',
+    'Attribute combinations criteria' => 'Critères de déclinaisons',
     'Clean selection' => 'Réinitialiser les champs',
     'Default category' => 'Catégorie par défaut',
     'ID' => 'ID',
+    'Minimum stock' => 'Stock minimum',
+    'Maximum stock' => 'Stock maximum',
+    'New' => 'Nouveau',
     'No' => 'Non',
     'Nothing selected' => 'Aucune sélection ...',
     'Price' => 'Prix',
     'Ref' => 'Référence',
+    'Sale' => 'Promo',
     'Search' => 'Rechercher',
+    'Search product' => 'Rechercher des produits',
```

Comment:
Pour éviter d'avoir 2 clés pour 1 traduction, hésite pas à chercher et virer les itérations inutiles (`product` sans S est utilisé uniquement dans menu-hook j'ai l'impression)